build_path.go

```diff
@@ -14,15 +14,30 @@ const KeyOpenAPITags = "openapi.tags"
 func buildPaths(ws *restful.WebService) spec.Paths {
 	p := spec.Paths{Paths: map[string]spec.PathItem{}}
 	for _, each := range ws.Routes() {
-		existingPathItem, ok := p.Paths[each.Path]
+		path := sanitizePath(each.Path)
+		existingPathItem, ok := p.Paths[path]
 		if !ok {
 			existingPathItem = spec.PathItem{}
 		}
-		p.Paths[each.Path] = buildPathItem(ws, each, existingPathItem)
+		p.Paths[path] = buildPathItem(ws, each, existingPathItem)
 	}
 	return p
 }
 
+func sanitizePath(restfulPath string) string {
```

Comment:
this clearly needs some comments about what the sanitation rules are + I would like to see unit tests for this function.